javaparser-core/src/main/java/com/github/javaparser/ast/body/ModifierSet.java

```diff
@@ -19,8 +19,11 @@
 package com.github.javaparser.ast.body;
 
 import com.github.javaparser.ast.AccessSpecifier;
+import com.github.javaparser.ast.Modifiers;
+import com.github.javaparser.ast.NodeWithModifiers;
 
 import java.lang.reflect.Modifier;
```

Comment:
"For the default keyword, it is even simpler: there is no modifier bit associated with the keyword. If a public, non-abstract, non-static method appears in an interface, it must be a default method. That’s how Method.isDefault() determines whether a method is a default method. Modifier.toString(…) has no chance of knowing whether the declaring class is an interface and therefore will never print default."

So the old interface based on ints cannot handle the default modifier.